compiler/qsc/src/interpret/package_tests.rs

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#![allow(clippy::needless_raw_string_hashes)]
+
+use crate::{interpret::Interpreter, packages::BuildableProgram};
+use indoc::indoc;
+use qsc_data_structures::{language_features::LanguageFeatures, target::TargetCapabilityFlags};
+use qsc_frontend::compile::SourceMap;
+use qsc_passes::PackageType;
+use qsc_project::{PackageGraphSources, PackageInfo};
+use rustc_hash::FxHashMap;
+
+#[test]
+fn import_and_call_reexport() {
+    let pkg_graph: PackageGraphSources = PackageGraphSources {
+        root: PackageInfo {
+            sources: vec![(
+                "PackageB.qs".into(),
+                indoc! {"
```

Comment:
nit: indentation in some of this test cases is a bit odd and makes it slightly harder to follow what the test case is doing.